docs/content/config/environment.md

```diff
@@ -168,7 +168,7 @@ Enabled by ENABLE_POSTFIX_VIRTUAL_TRANSPORT. Specify the final delivery of postf
 
 - **empty**: fail
 - `lmtp:unix:private/dovecot-lmtp` (use socket)
-- `lmtps:inet:<host>:<port>` (secure lmtp with starttls, take a look at <https://sys4.de/en/blog/2014/11/17/sicheres-lmtp-mit-starttls-in-dovecot/>)
```

Comment:
Out of curiosity, why remove the `<>`?

Reply:
I asked myself the opposite: why keeping them. They don't have a purpose + we don't use them in other places in our documentation where URLs are used.

Reply:
However: "Without angle brackets, the URL isn't converted into a link in many markdown parsers.". I've added them again.

Reply:
That's exactly what I figured as well :D Nice.